src/figuresClasses.ts

```diff
@@ -1,19 +1,119 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+function roundAreaToHundreths(area: number): number {
+  return Math.floor(area * 100) / 100;
 }
 
-export class Triangle {
-
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getInfo(): string;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
+
+  color: Color;
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(color: Color, a: number, b: number, c: number) {
```

Comment:
you should access those types through Figure, you're implementing figure interface - `Figure[typeName]`